tests/foreman/ui/test_smartclassparameter.py

```diff
@@ -1201,3 +567,12 @@ def test_positive_update_hidden_value_in_attribute(session, sc_params_list, modu
         ][0]
         assert host_matcher['Value']['value'] == host_override_value
         assert host_matcher['Value']['hidden'] is True
+        # Unhide
+        session.host.set_puppet_class_parameter_value(
+            module_host.name,
+            sc_param.parameter,
+            dict(overridden=True, hidden=False)
+        )
+        sc_parameter_values = session.sc_parameter.read(sc_param.parameter)
```

Comment:
try to avoid the whole read, if we are only using parameter tab   
```suggestion
        sc_parameter_values = session.sc_parameter.read(sc_param.parameter, widget_names='parameter')
```

Reply:
Hi @omkarkhatavkar, I'm not sure it would make any difference as there is only one tab under  Infrastructure > smart class parameters

Reply:
Oh if it is only one tab then it makes sense